frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.tsx

```diff
@@ -154,14 +158,17 @@ export const EmbedFrame = ({
     !!dashboard && isParametersWidgetContainersSticky(visibleParameters.length);
 
   const saveAsPDF = async () => {
-    const cardNodeSelector = `#${DASHBOARD_PDF_EXPORT_ROOT_ID}`;
-    await saveDashboardPdf(
-      cardNodeSelector,
-      name ?? t`Exported dashboard`,
-    ).then(() => {
-      // TODO: tracking
-      // trackExportDashboardToPDF(dashboard.id);
+    const dashboardAccessedVia = match(dashboard?.id)
+      .when(isJWT, () => "static-embed" as const)
+      .when(isUuid, () => "public-link" as const)
+      .otherwise(() => "sdk-embed" as const);
+
+    trackExportDashboardToPDF({
+      dashboardAccessedVia,
     });
+
+    const cardNodeSelector = `#${DASHBOARD_PDF_EXPORT_ROOT_ID}`;
+    await saveDashboardPdf(cardNodeSelector, name ?? t`Exported dashboard`);
```

Comment:
It's redundant to await here since nobody is awaiting `saveAsPDF()` anyway. It's passed to `onClick` which is only called synchronously.

With this, you can also remove `async` from the function declaration if needed.